src/shell/nushell.rs

```diff
@@ -0,0 +1,26 @@
+use super::shell::Shell;
+use std::path::Path;
+
+#[derive(Debug)]
+pub struct Nushell;
+
+impl Shell for Nushell {
+    fn to_clap_shell(&self) -> clap_complete::Shell {
+        panic!("Shell completion is not supported for Nushell (yet.)");
+    }
+
+    fn path(&self, path: &Path) -> anyhow::Result<String> {
+        let path = path
+            .to_str()
+            .ok_or_else(|| anyhow::anyhow!("Can't convert path to string"))?;
+        Ok(format!("let-env PATH = ($env.PATH | prepend {:?})", path))
```

Comment:
what's the difference between `let-env PATH` and `pathvar`?

* x-ref: https://github.com/Schniz/fnm/issues/463#issuecomment-1002651981

Reply:
I don't know - my nushell installation doesn't have `pathvar`

```
Error: nu::shell::external_command (link)

  × External command
   ╭─[entry #3:1:1]
 1 │ pathvar
   · ───┬───
   ·    ╰── can't run executable
   ╰────
  help: No such file or directory (os error 2)
```

[The nushell docs recommend this way to add to your `PATH`](https://www.nushell.sh/book/configuration.html#path-configuration)